### DISCUSSION TITLE
Can't create issue on mobile in github-app as it constantly opens in browser

### DISCUSSION THREAD

**Discussion by Der-Flip** [fast4x/RiMusic#4723] [General]
Is there a way this could be fixed? It requests to be opened externally in browser only for the browser to request to open in GitHub which instantly requests to open in browser and so on.

**Comment by fast4x** [comment]
Yep it's normal, it use device browser.

**Reply by Der-Flip** [reply]
ah, shame.... because keepass database was corrupted during syncing and I didn't have access to the original which rendered me unable to  make a bug report. For now, the bug is not happening anymore, but i didn't see it in the changelog fixed, so it might reappear... i'll wait and see then as I don't want to open a ticket for an out of date version on a bug I can't recreate now.
